datadog_checks_base/tests/base/checks/test_agent_check.py

```diff
@@ -1214,3 +1214,25 @@ def test_detect_typos_configuration_models(
         assert "Detected potential typo in configuration option" not in caplog.text
 
     assert typos == set(unknown_options)
+
+
+@pytest.mark.parametrize(
+    "integration_profiling",
+    ["true", "false"],
```

Comment:
```suggestion
    [True, False],
```
That way you can later just say `if integration_profiling:`